protocol/communities/community.go

```diff
@@ -972,6 +972,38 @@ func (o *Community) RemoveUserFromOrg(pk *ecdsa.PublicKey) (*protobuf.CommunityD
 	return o.config.CommunityDescription, nil
 }
 
+func (o *Community) RemoveSpecificUsersFromOrg(membersToRemove map[string]*protobuf.CommunityMember) *CommunityChanges {
```

Comment:
Just to keep the naming consistent.
```suggestion
func (o *Community) RemoveUsersFromOrg(membersToRemove map[string]*protobuf.CommunityMember) *CommunityChanges {
```


Also, I would love to see `RemoveUserFromOrg` and `RemoveUsersFromOrg` being compatible, that is:
```

func (o *Community) RemoveUserFromOrg(pk *ecdsa.PublicKey) (*CommunityChanges, error) {
    return o.RemoveUsersFromOrg([]*ecdsa.PublicKey{pk})
}
```

Do you think it is doable / worth the effort?

Reply:
I don't have a strong opinion here. From the code reading complexity, removeMemberFromOrg is much easier to read :) 
Not a problem to refactor

Reply:
refactored, changes in the separate commit